doc/apidoc/index.rst

```diff
@@ -17,6 +17,7 @@ Submodules
 * :ref:`Subplots: helper function for layout out multi-plot figures <subplots>`
 * :ref:`Figure Factories: helper methods for building specific complex charts <ff>`
 * :ref:`I/O: low-level interface for displaying, reading and writing figures <io>`
+* :mod:`plotly.colors`: colorscales and utility functions
```

Comment:
do we want to add `plotly.data` here too?

Reply:
yes